### ISSUE TITLE
Add option to automatically convert '*' to '\cdot' or '\times'

### ISSUE THREAD

**Issue by Mafrans** [joeyuping/quick_latex_obsidian#122]
Many standards for multiplication use the center dot (`\cdot`) or cross (`\times`) for multiplication of scalars, I suggest there should be an option to automatically convert the asterisk symbol (often used as a multiplication symbol) into one of these options, similar to how '/' automatically converts to `\frac`.

**Comment by joeyuping** [comment]
Hi!! 

The functionality can be achieved using the custom shorthand function of the plugin, by going to the plugin settings page, scroll to the last item, following the syntax as described in the description.

**Comment by joeyuping** [comment]
for the multiplication conversion, try adding the following to the custom shortcut setting:

*:::\cdot;

**Comment by Mafrans** [comment]
The custom shorthand works when a space is before/after the symbol (as the custom shortcuts are activated by pressing the spacebar) but typing f.e `a*b` will not result in `a \cdot b`, perhaps this is intended functionality, but it does make quick latex slightly less quick.

**Comment by asellingson28** [comment]
I would agree this should be an option in the custom quick latex settings.
